experiments/react-native-uifabric/src/components/RadioGroup/RadioGroup.tsx

```diff
@@ -82,7 +82,7 @@ export const RadioGroup = compose<IRadioGroupType>({
   },
   styles: {
     root: [],
-    label: [foregroundColorTokens, textTokens],
```

Comment:
doesn't the label still need text tokens?

Reply:
if the test page doesn't currently offer text customization, it seems worth adding a button to show it works; doesn't need to be exhaustive though.

Reply:
I talked with EJ and we thought we'd just allow the text color to change for now, which is why I got rid of textTokens. We'll probably add it in for V2.

Reply:
I believe as written if someone theme changes font family or anything else, this label won't respect the theme.  I don't expect people to make radio button-specific stylizations often, but we have to assign the text props coming from the theme which may be a higher-level customization.

Reply:
Yah you're right, I just added back textTokens in my latest iteration

Reply:
And I just tested in the test app, customizing the fontFamily, for instance, does change the label.